packages/cli/src/commands/start/startFrontend.ts

```diff
@@ -62,14 +62,30 @@ export async function startFrontend(options: StartAppOptions) {
   }
 
   const { name } = await fs.readJson(paths.resolveTarget('package.json'));
+  const config = await loadCliConfig({
+    args: options.configPaths,
+    fromPackage: name,
+    withFilteredKeys: true,
+  });
+
+  const appBaseUrl = config.frontendConfig.getString('app.baseUrl');
+  const backendBaseUrl = config.frontendConfig.getString('backend.baseUrl');
+  if (appBaseUrl === backendBaseUrl) {
+    console.log(
+      chalk.yellow(
+        `⚠️ Conflict between app baseUrl and backend baseUrl:
```

Comment:
Thinking it could be good to provide a built more guidance for the user, letting them know how to solve the issue. Thinking something along the lines of how the two configs are set to the same, but for local development that is unexpected since the backend is run separately. It is fixed by making sure that they each point to their respective local development ports.